dev/drivers/scripts/cam/stats/jevs_cam_nam_firewxnest_grid2obs_stats.sh

```diff
@@ -45,15 +45,15 @@ export job=${PBS_JOBNAME:-jevs_${MODELNAME}_${VERIF_CASE}_${STEP}}
 export jobid=$job.${PBS_JOBID:-$$}
 
 export COMIN=/lfs/h2/emc/vpppg/noscrub/$USER/${NET}/${evs_ver}
-export COMOUT=$COMIN/${STEP}/${COMPONENT}
+export COMOUT=/lfs/h2/emc/vpppg/noscrub/$USER/${NET}/${evs_ver}/${STEP}/${COMPONENT}
```

Comment:
With how COMOUT is defined in JEVS_CAM_STATS, COMOUT in the driver needs to be: 
```
export COMOUT=/lfs/h2/emc/vpppg/noscrub/$USER/${NET}/${evs_ver}
```

Reply:
Done.